app/views/state_file/questions/w2/edit.html.erb

```diff
@@ -44,7 +46,7 @@
       <% end %>
     </div>
 
-    <% if @box14_codes.present? %>
+    <% if @box14_codes.present? && current_state_code != 'nj' %>
```

Comment:
[boulder] would prefer to see a more dynamic approach to this rather than just checking state code. maybe it could look in the yml for the presence of any `_explanation_html` strings? bit of a pain but i think we're trying to avoid checking the state code as much as possible

Reply:
@jenny-heath refactored this, LMK what you think!